--- vars        2020-09-15 19:45:50.000000000 +0000
+++ vars.new	2020-09-15 19:53:33.503648351 +0000
@@ -84,7 +84,7 @@
 #   cn_only  - use just a CN value
 #   org      - use the "traditional" Country/Province/City/Org/OU/email/CN format
 
-#set_var EASYRSA_DN	"cn_only"
+set_var EASYRSA_DN	"org"
 
 # Organizational fields (used with 'org' mode and ignored in 'cn_only' mode.)
 # These are the default values for fields which will be placed in the
@@ -92,12 +92,12 @@
 # you may omit any specific field by typing the "." symbol (not valid for
 # email.)
 
-#set_var EASYRSA_REQ_COUNTRY	"US"
-#set_var EASYRSA_REQ_PROVINCE	"California"
-#set_var EASYRSA_REQ_CITY	"San Francisco"
-#set_var EASYRSA_REQ_ORG	"Copyleft Certificate Co"
-#set_var EASYRSA_REQ_EMAIL	"me@example.net"
-#set_var EASYRSA_REQ_OU		"My Organizational Unit"
+set_var EASYRSA_REQ_COUNTRY	"US"
+set_var EASYRSA_REQ_PROVINCE	"Oregon"
+set_var EASYRSA_REQ_CITY	"Corvallis"
+set_var EASYRSA_REQ_ORG	"Revolution Robotics, Inc."
+set_var EASYRSA_REQ_EMAIL	"ca@revolution-robotics.com"
+set_var EASYRSA_REQ_OU		"RevoEdge"
 
 # Choose a size in bits for your keypairs. The recommended value is 2048.  Using
 # 2048-bit keys is considered more than sufficient for many years into the
@@ -122,11 +122,11 @@
 
 # In how many days should the root CA key expire?
 
-#set_var EASYRSA_CA_EXPIRE	3650
+set_var EASYRSA_CA_EXPIRE	3650
 
 # In how many days should certificates expire?
 
-#set_var EASYRSA_CERT_EXPIRE	825
+set_var EASYRSA_CERT_EXPIRE	3650
 
 # How many days until the next CRL publish date?  Note that the CRL can still be
 # parsed after this timeframe passes. It is only used for an expected next
